app/services/users/services.rb

```diff
@@ -19,6 +19,7 @@ def initialize(user)
     def authorizations
       @list << BackendServices::RX if user.authorize :mhv_prescriptions, :access?
       @list << BackendServices::MESSAGING if user.authorize :mhv_messaging, :access?
+      @list << BackendServices::MEDICAL_RECORDS if user.authorize :mhv_medical_records, :access?
```

Comment:
Can you update the associated spec [here](https://github.com/department-of-veterans-affairs/vets-api/blob/aa17ab7a10e369f24f8acc7bbf59ac4e4dff2d59/spec/services/users/services_spec.rb#L11)?

Reply:
@dickdavis I'm not too familiar with this area. I've been using `BackendServices::MESSAGING` as a template for what to update, since it has exactly the same policy, but I don't see "messaging" being tested in `services_spec.rb`. Can you provide guidance on what is needed there?

Reply:
Looks like it isn't as simple as adding `medical-records` to the array, since the user must have an MHV account premium. Instead, you can add a scenario specifically for MHV users, like this:

```ruby
# frozen_string_literal: true

require 'rails_helper'

RSpec.describe Users::Services do
  describe '#authorizations' do
    subject { Users::Services.new(user).authorizations }

    let(:user) { build :user, :loa3 }

    it 'returns an array of services authorized to the initialized user', :aggregate_failures do
      expect(subject.class).to eq Array
      expect(subject).to match_array(
        %w[
          facilities
          hca
          edu-benefits
          evss-claims
          lighthouse
          form526
          user-profile
          appeals-status
          form-save-in-progress
          form-prefill
          identity-proofed
          vet360
        ]
      )
    end

    context 'with an loa1 user' do
      let(:user) { build :user }

      it 'returns only the services that are authorized to this loa1 user' do
        expect(subject).to match_array(
          %w[
            facilities
            hca
            edu-benefits
            user-profile
            form-save-in-progress
            form-prefill
          ]
        )
      end
    end

    # New scenario to cover medical-records
    context 'with an MHV Premium user' do
      let(:user) { build(:user, :mhv) }

      it 'returns an array including the medical-records service' do
        expect(subject).to include('medical-records')
      end
    end
  end
end
```